Assets/Scripts/Steam/SteamManager.cs

```diff
@@ -148,6 +150,20 @@ protected virtual void OnEnable()
                 m_SteamAPIWarningMessageHook = new SteamAPIWarningMessageHook_t(SteamAPIDebugTextHook);
                 SteamClient.SetWarningMessageHook(m_SteamAPIWarningMessageHook);
             }
+
+            if (m_AuthTicketForWebApiResponseCallback == null)
+            {
+                m_AuthTicketForWebApiResponseCallback = Callback<GetTicketForWebApiResponse_t>.Create(
+                    (GetTicketForWebApiResponse_t pCallback) =>
+                    {
+                        if (pCallback.m_eResult == EResult.k_EResultOK)
+                        {
+                            sessionTicketHandle = pCallback.m_hAuthTicket;
+                            sessionTicketString = System.BitConverter.ToString(pCallback.m_rgubTicket).Replace("-", "");
+                            Debug.LogWarning ("GetAuthTicketForWebApi succeed, try logging in again");
```

Comment:
Should this use one of our conditional print functions instead of Unity's logging directly?

Reply:
print() logs a log not a log warning.
I made this warning to stand out (warnings are yellow), so user knows to try again.
We decided that this two-step login will need to be changed.
So this line will probably disappear.